### PR TITLE
BLIP-2 blog post

### PR DESCRIPTION
**Pull request by MKhalusova**
This PR adds a blog post about the recently added BLIP-2 model. The initial google doc draft is [here](https://docs.google.com/document/d/1HdUzRTrFP3yuvDkiGNEdzn34hDfurfvIsr8DdjiYsOs/edit#).

### CODE REVIEW

File: blip-2.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+authors:
+- user: MariaK
+- user: JunnanLi
+---
+
+# Zero-shot image-to-text generation with BLIP-2
+
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
```
Comment (osanseviero) [inline blip-2.md:13]: Maybe add a ToC as in https://huggingface.co/blog/vision_language_pretraining

Comment (osanseviero) [inline blip-2.md:13]: Maybe add a colab as in https://huggingface.co/blog/image-similarity

**Review by osanseviero — APPROVED** [review]
Very cool! Thanks a lot

File: blip-2.md (hunk 1 above)
Comment (MKhalusova) [reply blip-2.md:13]: Added a ToC in 33f6619f47d7292951635fe70cf2832cb116f8f3

**Pull request merged**